src/pynxtools/nomad/schema.py

```diff
@@ -835,6 +836,7 @@ def init_nexus_metainfo():
     nexus_section = Section(
         validate=VALIDATE, name=__GROUPING_NAME, label=__GROUPING_NAME
     )
+    nexus_section.base_sections = [Schema.m_def]
```

Comment:
Just for future reference. Could you describe how this affects the metainfo generation?

Reply:
added